packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/Choices.js

```diff
@@ -28,30 +28,18 @@ const defaultTheme = {
     colors: {}
   }
 }
+
 export function Choices ({
   disabled = false,
   filteredChoiceIds = [],
+  focusedChoiceId = '',
   handleDelete = () => {},
+  handleFocusedChoice = () => {},
   onChoose = () => true,
   selectedChoiceIds = [],
   task,
   theme = defaultTheme
 }) {
-  const [focusIndex, setFocusIndex] = useState(null)
```

Comment:
I think you could have kept this to track state while the choices menu is open, but change the initial state from `null` to `focusedChoiceId` when the menu mounts.

Reply:
If you were to do that, then the Choice component should store the current choice when you press Not This or Identify, which then becomes the initial value here. Otherwise, this component wouldn’t need to change.

Reply:
I think I've refactored according to suggestions. There is now a `previousChoice` state in SurveyTaskContainer and `focusIndex` state in Choices. I was confused as I thought you were suggesting to move what was `focusedChoiceId` state from SurveyTaskContainer to Choices, but I think that interpretation of your suggestion is incorrect. Hopefully the current refactor better reflects your suggestion.